translations/japanese/core-commands.ts

```diff
@@ -0,0 +1,132 @@
+import type {Translations} from '../../server/chat';
+
+export const translations: Translations = {
+	strings: {
+		"Server version: <b>${version}</b>": "サーバーのバージョン: <b>${version}</b>",
+		"/mee - must not start with a letter or number": "文字や数字を最初においてはいけません。",
+		"What?! How are you not more excited to battle?! Try /battle! to show me you're ready.": "バトルが盛り上がってないって!?覚悟を決めて、/battle だ!",
+		"Access denied for custom avatar - make sure you're on the right account?": "このカスタムアバターを使用する権限がありません。許可されているアカウントでログインしてください。",
+		"Invalid avatar.": "アバター名が正しくありません。",
+		"Avatar changed to:": "アバターを次のものに変更しました。",
+		"Artist: ": "作者",
+		"No one has PMed you yet.": "誰もあなたにPMを送っていません。",
+		"You forgot the comma.": "コンマを忘れていませんか?",
+		"User ${targetUsername} not found. Did you misspell their name?": "ユーザー名:${targetUsername}は見つかりませんでした。名前を間違えていませんか?",
+		"User ${targetUsername} is offline.": "ユーザー名:${targetUsername}は見つかりませんでした。",
+		"The user \"${targetUsername}\" was not found.": "ユーザー名:\"${targetUsername}\"は見つかりませんでした。",
+		"The room \"${target}\" was not found.": "チャットルーム:\"${target}\"は見つかりませんでした。",
+		"You do not have permission to invite people into this room.": "あなたはこの部屋でユーザーを招待する権限を持っていません。",
+		"This user is already in \"${targetRoom.title}\".": "このユーザーはすでに\"${targetRoom.title}\"にいます。",
+		"Setting status messages in /busy is no longer supported. Set a status using /status.": "/busyでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"Setting status messages in /away is no longer supported. Set a status using /status.": "/awayでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"User '${target}' not found.": "ユーザー名'${target}'は見つかりませんでした。",
+		"${targetUser.name} does not have a status set.": "${targetUser.name}はステータスメッセージを設定していません。",
+		"${targetUser.name}'s status \"${targetUser.userMessage}\" was cleared by ${user.name}${displayReason}": "${targetUser.name}のステータスメッセージは${user.name}によって消去されました。${displayReason}",
+		"You don't have a status message set.": "ステータスメッセージを設定していません。",
+		"You have cleared your status message.": "ステータスメッセージを消去しました。",
+		"This user has not played any ladder games yet.": "このユーザーはladderが有効なバトルをしていません。",
+		"W[TN: initial for Wins]": "'Wins'の頭文字",
+		"L[TN: initial for Losses]": "'Losses'の頭文字",
```

Comment:
These are for columns with just a number in them; if it's possible to make these abbreviations shorter that would probably look better. If not then that's okay too; nothing will break, but the columns will be a bit wide.

Reply:
oh, I missed. Thank you!